MantidQt/CustomInterfaces/test/EnggDiffractionPresenterTest.h

```diff
@@ -1267,6 +1444,8 @@ class EnggDiffractionPresenterTest : public CxxTest::TestSuite {
   const static std::string g_eventModeRunNo;
   const static std::string g_validRunNo;
   const static std::string g_focusedRun;
+  const static std::string g_focusedBankFile;
+  const static std::string g_focusedFittingRunNo;
```

Comment:
Do these need to be `static` / `g_`? They could be `m_`. Not that it makes a big difference here, but `g_` will enforce a class-wide scope.